tico/utils/convert.py

```diff
@@ -162,6 +162,22 @@ def check_unsupported_target(exported_program: ExportedProgram):
         raise NotYetSupportedError("NOT SUPPORTED OPERATOR IN GRAPH MODULE")
 
 
+def check_training_ops(exported_program: ExportedProgram):
+    TRAINING_OPS = {
+        torch.ops.aten.dropout.default,
+        torch.ops.aten.native_dropout.default,
+    }
+    found = set()
+    for node in exported_program.graph.nodes:
+        if node.op == "call_function" and node.target in TRAINING_OPS:
+            found.add(node.target)
+
+    if found:
+        raise RuntimeError(
+            f"Detected training-mode ops {sorted(found)}. Call `model.eval()` before export."
```

Comment:
RuntimeError will the whole list of training ops. It is the reason we need for-loop.